### ISSUE TITLE
Redefine .CAPTURE options

### ISSUE THREAD

**Issue by Araneidae** [PandABlocks/PandABlocks-server#23]
At present the `.CAPTURE` attribute for `pos_out` fields is an enum with the following slightly arbitrary list of values: "No", "Value", "Diff", "Sum", "Mean", "Min", "Max", "Min Max", "Min Max Mean".

It would be straightforward to change this attribute to simply take a list of keywords (in this case just "Value", "Diff", "Sum", "Min", "Max", "Mean", probably to be extended with "StdDev" as part of issue #13).  The main impact would be that `*ENUMS.block.field.CAPTURE?` would no longer work, and we'd probably need an extra command to interrogate the list of available keywords.

Keyword options would be normalised into a fixed order with duplicates removed, so a typical exchange might look like this:
```
< BLOCK.FIELD.CAPTURE=Min Min Value Diff Value
> OK
< BLOCK.FIELD.CAPTURE?
> OK =Value Diff Min
```

Questions

- Do we allow "No" as a reported value or just report an empty string?  If we don't need to preserve compatibility I'd suggest dropping "No" as a valid value, and report an empty string for disabled capture, but there's no compelling need to do this.
- What shall we call the command to interrogate capture keywords?  How about `*PCAP.CAPTURE_OPTIONS?`?

@thomascobb, I think this change could be a useful preliminary for #13.

**Comment by thomascobb** [comment]
Personally, I'm happy with the slightly arbitrary list of values. We have some clear use cases for the selection we have at the moment, so adding "Mean StdDev" and "Min Max Mean StdDev" would cater for everything I can think of

**Comment by Araneidae** [comment]
Implemented in commit 231a067bc298df5bc12baa21ab467a791885d9ea
